packages/cli/src/create/data-set/DataSet.handler.ts

```diff
@@ -10,13 +10,70 @@
  */
 
 import type { IHandlerParameters } from "@zowe/imperative";
+import { CreateDefaults } from "../CreateDefaults";
 import type { ZSshClient, ds } from "zowe-native-proto-sdk";
+import type { DatasetAttributes } from "zowe-native-proto-sdk/src/doc/gen/common";
```

Comment:
We should probably avoid accessing things from the `src` directory of a package 😋